DataMapper/main.bal

```diff
@@ -0,0 +1,76 @@
+type Contact record {
+    string email;
+    string phone_number;
+};
+
+type OrderData record {
+    string order_item_id;
+    string product_id;
+    int quantity;
+    string customer_id;
+};
+
+type InventoryData record {
+    string product_id;
+    string name;
+    string description;
+    string category;
+    decimal price;
+    int stock_quantity;
+    string manufacturer;
+    string release_date;
+};
+
+type Customer record {
+    string name;
+    string email;
+    string phone_number;
+};
+
+type Shipping_address record {
```

Comment:
```suggestion
type ShippingAddress record {
```

Reply:
These records and their fields were generated by the Data mapper using this format.

Reply:
Changed the keys of the json files, and these got updated.